conformant = "Partial"
notes = """
Does not detect type violation for some deeply-nested types.
Does not properly handle `|` for unions in some recursive type alias definitions.
Does not detect cyclical references in recursive type alias definition.
"""
output = """
File "aliases_recursive.py", line 19, in <module>: Type annotation for j4 does not match type of assignment [annotation-type-mismatch]
File "aliases_recursive.py", line 20, in <module>: Type annotation for j5 does not match type of assignment [annotation-type-mismatch]
File "aliases_recursive.py", line 38, in <module>: Type annotation for t6 does not match type of assignment [annotation-type-mismatch]
File "aliases_recursive.py", line 39, in <module>: Type annotation for t6 does not match type of assignment [annotation-type-mismatch]
File "aliases_recursive.py", line 50, in <module>: Type annotation for m7 does not match type of assignment [annotation-type-mismatch]
File "aliases_recursive.py", line 51, in <module>: Type annotation for m8 does not match type of assignment [annotation-type-mismatch]
File "aliases_recursive.py", line 62, in <module>: unsupported operand type(s) for |: ''GenericTypeAlias1[T1]': str' and 'T1: TypeVar' [unsupported-operands]
File "aliases_recursive.py", line 63, in <module>: Invalid type annotation 'list[str]'  [invalid-annotation]
File "aliases_recursive.py", line 66, in <module>: Invalid type annotation 'list[str]'  [invalid-annotation]
File "aliases_recursive.py", line 66, in <module>: Invalid type annotation 'GenericTypeAlias1[str]'  [invalid-annotation]
File "aliases_recursive.py", line 67, in <module>: Invalid type annotation 'list[str]'  [invalid-annotation]
File "aliases_recursive.py", line 67, in <module>: Invalid type annotation 'GenericTypeAlias1[str]'  [invalid-annotation]
File "aliases_recursive.py", line 69, in <module>: unsupported operand type(s) for |: ''GenericTypeAlias2[T1, T2]': str' and 'T1: TypeVar' [unsupported-operands]
File "aliases_recursive.py", line 71, in <module>: Invalid type annotation 'list[str, int]'  [invalid-annotation]
File "aliases_recursive.py", line 71, in <module>: Invalid type annotation 'GenericTypeAlias2[str, int]'  [invalid-annotation]
File "aliases_recursive.py", line 72, in <module>: Invalid type annotation 'list[str, float]'  [invalid-annotation]
File "aliases_recursive.py", line 72, in <module>: Invalid type annotation 'GenericTypeAlias2[str, float]'  [invalid-annotation]
File "aliases_recursive.py", line 73, in <module>: Invalid type annotation 'list[str, int]'  [invalid-annotation]
File "aliases_recursive.py", line 73, in <module>: Invalid type annotation 'GenericTypeAlias2[str, int]'  [invalid-annotation]
"""
conformance_automated = "Fail"
errors_diff = """
Line 67: Expected 1 errors, got 2 (['File "aliases_recursive.py", line 67, in <module>: Invalid type annotation \\'list[str]\\'  [invalid-annotation]', 'File "aliases_recursive.py", line 67, in <module>: Invalid type annotation \\'GenericTypeAlias1[str]\\'  [invalid-annotation]'])
Line 73: Expected 1 errors, got 2 (['File "aliases_recursive.py", line 73, in <module>: Invalid type annotation \\'list[str, int]\\'  [invalid-annotation]', 'File "aliases_recursive.py", line 73, in <module>: Invalid type annotation \\'GenericTypeAlias2[str, int]\\'  [invalid-annotation]'])
Line 76: Expected 1 errors
Line 78: Expected 1 errors
Line 38: Unexpected errors ['File "aliases_recursive.py", line 38, in <module>: Type annotation for t6 does not match type of assignment [annotation-type-mismatch]']
Line 39: Unexpected errors ['File "aliases_recursive.py", line 39, in <module>: Type annotation for t6 does not match type of assignment [annotation-type-mismatch]']
Line 50: Unexpected errors ['File "aliases_recursive.py", line 50, in <module>: Type annotation for m7 does not match type of assignment [annotation-type-mismatch]']
Line 51: Unexpected errors ['File "aliases_recursive.py", line 51, in <module>: Type annotation for m8 does not match type of assignment [annotation-type-mismatch]']
Line 62: Unexpected errors ['File "aliases_recursive.py", line 62, in <module>: unsupported operand type(s) for |: \\'\\'GenericTypeAlias1[T1]\\': str\\' and \\'T1: TypeVar\\' [unsupported-operands]']
Line 63: Unexpected errors ['File "aliases_recursive.py", line 63, in <module>: Invalid type annotation \\'list[str]\\'  [invalid-annotation]']
Line 66: Unexpected errors ['File "aliases_recursive.py", line 66, in <module>: Invalid type annotation \\'list[str]\\'  [invalid-annotation]', 'File "aliases_recursive.py", line 66, in <module>: Invalid type annotation \\'GenericTypeAlias1[str]\\'  [invalid-annotation]']
Line 69: Unexpected errors ['File "aliases_recursive.py", line 69, in <module>: unsupported operand type(s) for |: \\'\\'GenericTypeAlias2[T1, T2]\\': str\\' and \\'T1: TypeVar\\' [unsupported-operands]']
Line 71: Unexpected errors ['File "aliases_recursive.py", line 71, in <module>: Invalid type annotation \\'list[str, int]\\'  [invalid-annotation]', 'File "aliases_recursive.py", line 71, in <module>: Invalid type annotation \\'GenericTypeAlias2[str, int]\\'  [invalid-annotation]']
Line 72: Unexpected errors ['File "aliases_recursive.py", line 72, in <module>: Invalid type annotation \\'list[str, float]\\'  [invalid-annotation]', 'File "aliases_recursive.py", line 72, in <module>: Invalid type annotation \\'GenericTypeAlias2[str, float]\\'  [invalid-annotation]']
"""
